x-pack/plugins/drilldowns/public/components/list_manage_drilldowns/list_manage_drilldowns.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiButton,
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiSpacer,
+  EuiTextColor,
+} from '@elastic/eui';
+import React, { useState } from 'react';
+import {
+  txtCreateDrilldown,
+  txtDeleteDrilldowns,
+  txtEditDrilldown,
+  txtSelectDrilldown,
+} from './i18n';
+
+export interface DrilldownListItem {
+  id: string;
+  actionName: string;
+  drilldownName: string;
+  icon?: string;
+}
+
+export interface ListManageDrilldownsProps {
+  drilldowns: DrilldownListItem[];
+
+  onEdit?: (id: string) => void;
+  onCreate?: () => void;
+  onDelete?: (ids: string[]) => void;
+}
+
+const noop = () => {};
+
+export const TEST_SUBJ_DRILLDOWN_ITEM = 'list-manage-drilldowns-item';
+
+export function ListManageDrilldowns({
+  drilldowns,
+  onEdit = noop,
```

Comment:
Is there a use case for using this without edit/create/delete fns?  I would have thought they'd be required.  or maybe undefined if not required and then you hide the buttons.

Reply:
I don't think there is a valid use case for not passing those.
But keeping those optional with this `noop` allows to have storybook setup and unit tests a bit cleaner. 

e.g. setup for unit testing `delete` fn would be just
```tsx
const onDelete = jest.fn();
render(<ListManageDrilldowns onDelete={onDelete} />)
```

Reply:
Can you create a helper function (in a test folder) to help the storybook setup be cleaner? If there is not a reason for prod code to leave those out, I don't think we should allow it only for the sake of making writing tests easier.

Reply:
The component code also becomes a bit less lean because of branching like.
```
if (onDelete) {
 onDelete()
}
```

I don't think it is a big problem that consumer of component won't be shouted at if no callbacks are passed. I doubt this could be missed during testing